File: src/main/java/org/customer/controller/AccountViewController.java
package org.customer.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/accounts")
public class AccountViewController {

    @GetMapping("/open")
    public String openAccountForm() {
        return "accounts"; // Refers to accounts.html in /templates/
    }
}

File: src/main/java.kotlin/org/customer/controller/AccountViewController.kt
package org.customer.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/accounts")
class AccountViewController {
    @GetMapping("/open")
    fun openAccountForm(): String {
        return "accounts" // Refers to accounts.html in /templates/
    }
}
